classlib/src/main/java/org/teavm/classlib/java/lang/TBoolean.java

```diff
@@ -97,11 +88,11 @@ public static boolean getBoolean(String key) {
     }
 
     public static boolean logicalAnd(boolean a, boolean b) {
-        return a & b;
+        return a && b;
```

Comment:
What's the purpose of this change?

Reply:
Correspondence of name to the implementation.

Reply:
But in this particular context what's the difference between & and &&?

Reply:
No difference but short-circuiting. Actually this can be reverted if needed.